README.md

```diff
@@ -27,34 +34,67 @@ cd $GOPATH/github.com/src-d/code-annotation
 make serve
 ```
 
-## Development
+## Importing and Exporting Data
 
-Backend:
+### Import File Pairs for Annotation
 
-```
-make gorun
+The file pairs must be initially provided via a database, either [SQLite](https://sqlite.org/) or [PostgreSQL](https://www.postgresql.org/). The database should contain rows of file pairs in the following table:
+
+```SQL
+CREATE TABLE files (
+    blob_id_a TEXT, repository_id_a TEXT, commit_hash_a TEXT, path_a TEXT, content_a TEXT,
+    blob_id_b TEXT, repository_id_b TEXT, commit_hash_b TEXT, path_b TEXT, content_b TEXT,
+    score DOUBLE PRECISION);
```

Comment:
To keep README concise and maintainable, I suggest to remove DDL from here and just link to the relevant place in FS/code.

Reply:
sorry, can you point to such place? It's expected input schema. Not schema created by our tool.
Or do you suggest to use `cli/examples/import/example.sql`?

Reply:
I think it is relevant to document clearly the expected input DB schema that must be created by the users of the app.
This line is not a reference for developers, it is actually an important required first step, without this the annotation tool will have no data to work with.

Reply:
The documentation of the expected database, `dbutil` expected formats... and other internal things could be in a separated Documentation file, linked from the main `README.md`

Reply:
I'm not suggesting to avoid documenting it, for sure; I'm just suggesting to document separately, not in the `README.md` that should be only a first and quick contact with the app.

Reply:
I agree with @carlosms 
it's necessary information to start using the app. It should be in README.

Reply:
From the [`README.md` guidelines](https://github.com/src-d/guide/blob/master/engineering/documentation.md#readmemd)
>_[...] If the instructions are long they should be part of a different document linked from this one_ 

>_[...] This document also serves as the index for all other documentation_

The current "dbutils" documentation is double the size the rest of the current `README.md`, what could be considered as a case of "_If the instructions are long_"
I'm not sure if @campoy could help us to understand if this is a case of "_too large install instructions_"

On the other hand, this app is a tool to annotate project, so I think that it could be wrong to focus more on how to install rather than in how to use it and what to expect from it.

Reply:
from my point of view this section is https://github.com/src-d/guide/blob/master/engineering/documents/README.tmpl.md#example-of-utilization in template

Reply:
I think the readme is not that big and spreading sections in different files can make it difficult to find things. But I guess it's up to @bzz.

What I am strongly against is removing documentation and pointing to source code.

Reply:
Ok, @carlosms you have the point in https://github.com/src-d/code-annotation/pull/29#discussion_r163819521


I agree, but If possible I would prefer to have a link in REAME to the https://github.com/src-d/code-annotation/blob/master/cli/examples/import/example.sql#L6 as from what I can see - it contains the very same DDL.  

In case of future changes - there is no need to maintain it in two different places, which has a benefit of avoiding documentation rot.

Reply:
Yes, that can also work. even if it's not my favourite approach (personal preference). I'll make the change right now.

Reply:
@carlosms I might be missing something, but I think I have already saw it being replaced by the link to the file and now it's SQL in README here again here :/

Reply:
my bad. I fetched a wrong origin branch during rebase.